src/utils/EigenIO.hpp

```diff
@@ -24,3 +24,8 @@ inline Eigen::IOFormat debug()
 } // namespace eigenio
 } // namespace utils
 } // namespace precice
+
+template <>
```

Comment:
Shouldn't this be defined in the [`utils/fmtEigen.hpp`](https://github.com/precice/precice/blob/develop/src/utils/fmtEigen.hpp) ?